test/specs/tests.with.po.js

```diff
@@ -0,0 +1,54 @@
+const { expect } = require('chai');
+const { pages } = require('../pages/Pages');
+
+describe('Hometask test suite with PO', () => {
+    it('Perform sorting on the Inventory page', async () => {
+        await pages.loginPage.navigate();
+        await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+        await pages.inventoryPage.sortItems();
+
+        expect(await pages.inventoryPage.getCurrentSortingOption()).to.be.equal('Price (high to low)');
+        
+        let sortedPrices = await pages.inventoryPage.getPrices()
+        let sortedPricesCheck = [...sortedPrices].sort((a, b) => b - a);
+        
+        expect(sortedPrices).to.deep.equal(sortedPricesCheck);
+    });
+
+    it('Add random products to the cart', async () => {
+        await pages.loginPage.navigate();
+        await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+        let itemsId = await pages.inventoryPage.getItems();
+        let randomItemId0 = Math.floor(Math.random() * itemsId.length);
+        let randomItemId1 = Math.floor(Math.random() * itemsId.length);
```

Comment:
This is not rock solid approach due to next things:
1. It's possible that second item will be the same as first
2. The idea was to practise adding random amount of random items. 
I'd rather implement a method which will add random amount of random items to the cartt and collect all the necessary items data